error/src/lib.rs

```diff
@@ -400,6 +400,10 @@ impl CaliptraError {
     pub const RUNTIME_INCREMENT_PCR_RESET_MAX_REACHED: CaliptraError =
         CaliptraError::new_const(0x000E003A);
 
+    /// PCR Runtime Errors
+    pub const RUNTIME_PCR_RESERVED: CaliptraError = CaliptraError::new_const(0x000E0015);
```

Comment:
These errror codes are already used by other runtime errors. Can you make them 0x000E003B and 0x000E003C respectively?